presto-main/src/main/java/com/facebook/presto/cost/HistoryBasedPlanStatisticsManager.java

```diff
@@ -51,7 +52,7 @@ public HistoryBasedPlanStatisticsManager(ObjectMapper objectMapper, SessionPrope
         requireNonNull(objectMapper, "objectMapper is null");
         this.sessionPropertyManager = requireNonNull(sessionPropertyManager, "sessionPropertyManager is null");
         this.historyBasedStatisticsCacheManager = new HistoryBasedStatisticsCacheManager();
-        ObjectMapper newObjectMapper = objectMapper.copy().configure(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS, true);
+        ObjectMapper newObjectMapper = objectMapper.copy().configure(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS, true).configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true);
```

Comment:
is this still needed once you remove the annotations that shouldn't have been there?

Reply:
If this is to future-proof serialization between java and native plans wdyt about adding a test (or test framework) that compares the serialized plans?